### PR TITLE
Wsdt/erigon

### PR DESCRIPTION
**Pull request by wsdt**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Add Erigon to repository as part of bedrock folder. 

## Changes

- Add Erigon submodule
- Add Github Action that automatically updates from our forked Erigon repository
- Erigon repository (also creates a new PR if there are changes from the original repository)

## Testing

No automated tests.

### CODE REVIEW

File: .github/workflows/sync-submodules.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        git config --global user.email '<>'
+        git pull origin devel
+        git pull --tags
+        cd ../..
+        git add .
+        git commit -m "Updated Erigon submodule" || echo "Nothing to commit"
+        git push || echo "Nothing to push"
+        
+        #git pull --recurse-submodules
+        #git submodule update --remote --recursive
```
Comment (souradeep-das) [inline .github/workflows/sync-submodules.yml:57]: do we want to create a PR here?

**Review by souradeep-das — APPROVED** [review]
lgtm

File: .github/workflows/sync-submodules.yml (hunk 1 above)
Comment (InoMurko) [reply .github/workflows/sync-submodules.yml:57]: yeah!

**Comment by InoMurko** [comment]
goooooooddddd

File: .github/workflows/sync-submodules.yml (hunk 1 above)
Comment (wsdt) [reply .github/workflows/sync-submodules.yml:57]: done thx!

**Review by InoMurko — APPROVED** [review]

**Pull request merged**